docs/reference/quadratic.rst

```diff
@@ -215,3 +223,81 @@ Converting to and from other data structures:
    from_networkx_graph
 
 See also: :ref:`serialization functions<serialization_dimod>`
+
+.. _dqm:
+
+Discrete Quadratic Models
+=========================
+
+:term:`Discrete quadratic model`\ s (DQMs) are described under
+:std:doc:`Discrete Quadratic Models <oceandocs:concepts/dqm>`.
+
+DQM Class
+---------
+
+.. currentmodule:: dimod
+
+.. autoclass:: DiscreteQuadraticModel
+
+Attributes
+~~~~~~~~~~
+
+.. autosummary::
+   :toctree: generated/
+
+   ~DiscreteQuadraticModel.adj
+   ~DiscreteQuadraticModel.variables
+
+Methods
+~~~~~~~
+
+.. autosummary::
+   :toctree: generated/
+
+   ~DiscreteQuadraticModel.add_linear_equality_constraint
+   ~DiscreteQuadraticModel.add_variable
+   ~DiscreteQuadraticModel.copy
+   ~DiscreteQuadraticModel.from_file
```

Comment:
While we're at it, this is missing `energy` and `energies`...

Reply:
Added